openassessment/templates/openassessmentblock/leaderboard/oa_leaderboard_show.html

```diff
@@ -2,27 +2,26 @@
 {% spaceless %}
 <li id="openassessment__leaderboard__{{ xblock_id }}" class="openassessment__steps__step step--leaderboard is--complete ui-slidable__container is--showing {% if allow_latex %} allow--latex{% endif %}" tabindex="-1">
     <header class="step__header ui-slidable__control">
-        <h2 class="step__title">
+        <h4 class="step__title">
             <button class="ui-slidable" aria-expanded="true" id="oa_leaderboard_{{ xblock_id }}" aria-controls="oa_leaderboard_{{ xblock_id }}_content">
                 <span class="wrapper--copy">
                     <span class="step__label">{% trans "Top Responses" %} </span>
                 </span>
             </button>
-        </h2>
+        </h4>
     </header>
     <div class="wrapper--step__content ui-slidable__content" aria-labelledby="oa_leaderboard_{{ xblock_id }}" id="oa_leaderboard_{{ xblock_id }}_content">
-        <h3 class="leaderboard__title">{% trans "Top-Scoring Responses for This Assignment" %}</h3>
         <ol class="list leaderboard__score__list">
         {% for topscore in topscores %}
             <li class="leaderboard__score__item">
-                <h4 class="leaderboard__list__number">{{ forloop.counter }}</h4>
+                <div class="leaderboard__list__number">{{ forloop.counter }}</div>
                 {% with num_points=topscore.score %}
-                <h4 class="leaderboard__score__title">
+                <div class="leaderboard__score__title">
                     {% blocktrans %}{{ num_points }} points{% endblocktrans %}
-                </h4>
+                </div>
                 {% endwith %}
                 <div class="leaderboard__answer">
-                    {% trans "Your peer's response to the question above:" as translated_label %}
```

Comment:
Can you provide a link to a problem on your sandbox with the leaderboard enabled?